### PR TITLE
8 serialize dataprocess object

### PR DESCRIPTION
**Pull request by arielleleon**
Add aind-log-utils and serialize dataproc

### CODE REVIEW

File: metadata/metadata.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 metadata_version: 1
-name: aind-ophys-dff
+name: LOGGING aind-ophys-dff
```
Comment (j-friedrich) [inline metadata/metadata.yml:2]: 😂

File: code/dff.py [hunk 2]
```diff
@@ -1,63 +1,82 @@
...
 ) -> None:
     """Writes output metadata to plane processing.json
 
     Parameters
     ----------
     metadata: dict
         parameters from suite2p motion correction
-    input_fp: str
-        path to data input
+    raw_movie: str
```
Comment (j-friedrich) [inline code/dff.py:30]: argument name is input_fp, not raw_movie. And the input is not a raw movie, but (neuropil-corrected) traces

File: code/dff.py [hunk 3]
```diff
@@ -1,63 +1,82 @@
...
-                    code_url=(os.getenv("DFF_EXTRACTION_URL")),
-                    parameters=metadata,
-                )
-            ],
-        )
+    data_proc = DataProcess(
+        name=ProcessName.DF_F_ESTIMATION,
+        software_version=os.getenv("VERSION", ""),
+        start_date_time=start_time.isoformat(),
+        end_date_time=end_time.isoformat(),
```
Comment (j-friedrich) [inline code/dff.py:45]: is the time format consistent between capsules? You don't seem to use `.isoformat()` in some of the others? Maybe it doesn't matter cause aind-data-schema reformats it consistently?

**Review by j-friedrich — APPROVED** [review]

File: code/dff.py (hunk 3 above)
Comment (j-friedrich) [reply code/dff.py:45]: Oh, I overlooked it in the oasis caspule, you likely have the `.isoformat()` in every caspule

File: metadata/metadata.yml (hunk 1 above)
Comment (arielleleon) [reply metadata/metadata.yml]: omg, how....

File: environment/Dockerfile [hunk 4]
```diff
@@ -19,5 +19,5 @@ RUN pip3 install -U --no-cache-dir \
 
 RUN pip3 install -U --no-cache-dir \
     aind-ophys-utils==0.0.7 \
-    aind-data-schema==0.22.0 \
-    pydantic==2.8.2
```
Comment (seanmcculloch) [inline environment/Dockerfile:23]: Re: removing pydantic
aind-data-schema depends on pydantic>=2.7, so it is not necessary to include as a dependency here, but all other capsules in the pipeline include it.

**Review by seanmcculloch — APPROVED** [review]

**Pull request merged**